Cargo.toml

```diff
@@ -41,6 +42,7 @@ bytes = { default-features = false, version = "1.0" }
 criterion = { default-features = false, version = "0.3" }
 csv = "1"
 futures = { default-features = false, version = "0.3" }
+rand = { default-features = false, features = ["getrandom"], version = "0.8" }
```

Comment:
It is unfortunate that cargo forces the declaration of the same dependencies with different features.

For downstream libraries, the use of `rand_core` implies the addition of a possibly unused dependency in a production environment.
For development, `getrandom` is one of the methods available to generate entropy.